scipy/special/add_newdocs.py

```diff
@@ -3392,7 +3392,7 @@ def add_newdoc(name, doc):
     r"""
     gammainc(a, x)
 
-    Regularized lower incomplete gamma function.
+    Regularized lower incomplete Gamma function.
```

Comment:
`gamma ` normally written lowercase I believe. For example:
https://math.stackexchange.com/questions/1363278/does-the-definition-of-the-upper-case-incomplete-gamma-function-gammaa-x-in
https://en.wikipedia.org/wiki/Gamma_function

Could you undo these changes?

Reply:
Sorry about that, no idea why I changed it to `Gamma`.